tools/vpt/vpt_segment.xml

```diff
@@ -0,0 +1,940 @@
+<tool id="vpt_segment" name="Vizgen Post-processing Tool - Segment Cells" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@">
+    <description>Reprocess and refine the single-cell results of MERSCOPE experiments</description>
```

Comment:
name / description is a bit too long ... can you shorten it maybe? The word tool is not needed imho.

"reprocess and refine" can maybe also be shortend

```suggestion
    <description>Refine the results of MERSCOPE experiments</description>
```